doc/changelog.dox.in

```diff
@@ -6,6 +6,7 @@
 
 ### PDL Tools 1.5 (packaged with SUgAR 1.0) -- Current version
 - Modifications in preparation to make PDLTools public. Removing module with pending IP (connected components).
+- Add module one_vs_rest for multi-class classification
```

Comment:
Update the section title to the following:
### PDL Tools 1.5dev (packaged with SUgAR 1.0) -- Current version
Then add your module to it (ex: "Add module one_vs_rest for multi-class classification"), as your CL will technically go into a future release not 1.5.